Doc/library/re.rst

```diff
@@ -1565,21 +1565,28 @@ search() vs. match()
 
 .. sectionauthor:: Fred L. Drake, Jr. <fdrake@acm.org>
 
-Python offers two different primitive operations based on regular expressions:
-:func:`re.match` checks for a match only at the beginning of the string, while
-:func:`re.search` checks for a match anywhere in the string (this is what Perl
-does by default).
+Python offers different primitive operations based on regular expressions:
+
++ :func:`re.search` checks for a match anywhere in the string,
+  (this is what Perl does by default)
++ :func:`re.fullmatch` checks for entire string to be a match
++ :func:`re.match` checks for a match only at the beginning of the string
```

Comment:
Please leave these in the order of `match`, then `search`, then `fullmatch`. That way the match the examples immediately below.